localstack/services/s3/provider.py

```diff
@@ -585,6 +597,37 @@ def delete_bucket_lifecycle(
         store = self.get_store()
         store.bucket_lifecycle_configuration.pop(bucket, None)
 
+    def put_bucket_cors(
+        self,
+        context: RequestContext,
+        bucket: BucketName,
+        cors_configuration: CORSConfiguration,
+        content_md5: ContentMD5 = None,
+        checksum_algorithm: ChecksumAlgorithm = None,
+        expected_bucket_owner: AccountId = None,
+    ) -> None:
+        response = call_moto(context)
+        # max 100 rules
+        # validate CORS? see moto
```

Comment:
Leftover as well, moto was already validating the CORS rules. I just had to add a patch because it was not handling some cases. Will be removed as well! 👍 thanks for catching this